rust_src/src/character.rs

```diff
@@ -1,12 +1,46 @@
 //! Operations on characters.
 
+use libc::{c_uchar, ptrdiff_t};
+
 use remacs_macros::lisp_fn;
 use remacs_sys::EmacsInt;
 
 use lisp::LispObject;
 use lisp::defsubr;
 use multibyte::{make_char_multibyte, raw_byte_from_codepoint_safe};
 use multibyte::MAX_CHAR;
+use threads::ThreadState;
+
+/// True iff byte starts a character in a multibyte form.
+///
+/// Same as the `CHAR_READ_P` macro.
```

Comment:
Same as `CHAR_HEAD_P` ?

Reply:
yes. `read` is a typo in the comment and the function name.